__tests__/LocationSampler.js

```diff
@@ -0,0 +1,105 @@
+import 'react-native';
+import { NativeModules } from "react-native";
+import React from 'react';
+import LocationSampler from '../built/LocationSampler.js';
+const ReactNativeFS = require("react-native-fs");
+
+import renderer from 'react-test-renderer';
+
+describe('the constructor', () => {
+    test('does not set negative sampling intervals', () => {
+        let locationSampler = new LocationSampler(-1000, false, "Test");
+        expect(locationSampler.interval).toEqual(locationSampler.DEFAULT_INTERVAL)
+    });
+    test('does set positive sampling intervals', () => {
+        let locationSampler = new LocationSampler(1234, false, "Test");
+        expect(locationSampler.interval).toEqual(1234)
+    });
+    test('does not set running to true instantiation', () => {
```

Comment:
I agree with this, I've removed all constructor tests, except the positive/negative location sampling interval.